### PR TITLE
Fix missing word

### PR DESCRIPTION
**Pull request by TMoMoreau**
Third time's a charm. Added the missing word, few other minor changes. Issue #1316

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Review by ElPaisano — APPROVED** [review]
Tested links and reviewed - looks good

**Comment by ElPaisano** [comment]
@TMoMoreau looks like this is two commits behind main, so good idea to merge those in first before merging this into main

File: docs/concepts/ipns.md [hunk 1]
```diff
@@ -5,23 +5,25 @@ description: Learn about mutability in IPFS, InterPlanetary Name System (IPNS),
...
+      - [IPNS over PubSub](#ipns-over-pubsub)
+        - [Publishing IPNS records over PubSub lifecycle](#publishing-ipns-records-over-pubsub-lifecycle)
+    - [Tradeoffs between consistency vs. availability](#tradeoffs-between-consistency-vs-availability)
+      - [IPNS record validity](#ipns-record-validity)
+      - [Practical considerations](#practical-considerations)
+  - [IPNS in practice](#ipns-in-practice)
+    - [Resolving IPNS names using IPFS gateways](#resolving-ipns-names-using-ipfs-gateways)
+    - [Publishing IPNS names](#publishing-ipns-names)
+  - [Alternatives to IPNS](#alternatives-to-ipns)
+  - [Further Resources](#further-resources)
```
Comment (Jorropo) [inline docs/concepts/ipns.md:26]:
```suggestion- [Mutability in IPFS](#mutability-in-ipfs)
- [How IPNS works](#how-ipns-works)
  - [Anatomy of an IPNS name](#anatomy-of-an-ipns-name)
    - [How IPNS names relate to content paths](#how-ipns-names-relate-to-content-paths)
  - [IPNS names are self-certifying](#ipns-names-are-self-certifying)
  - [Common IPNS operations](#common-ipns-operations)
  - [IPNS is transport agnostic](#ipns-is-transport-agnostic)
    - [IPNS over the DHT](#ipns-over-the-dht)
    - [IPNS over PubSub](#ipns-over-pubsub)
      - [Publishing IPNS records over PubSub lifecycle](#publishing-ipns-records-over-pubsub-lifecycle)
  - [Tradeoffs between consistency vs. availability](#tradeoffs-between-consistency-vs-availability)
    - [IPNS record validity](#ipns-record-validity)
    - [Practical considerations](#practical-considerations)
- [IPNS in practice](#ipns-in-practice)
  - [Resolving IPNS names using IPFS gateways](#resolving-ipns-names-using-ipfs-gateways)
  - [Publishing IPNS names](#publishing-ipns-names)
- [Alternatives to IPNS](#alternatives-to-ipns)
```

**Review by Jorropo — APPROVED** [review]
LGTM

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Pull request merged**